presto-docs/src/main/sphinx/release/release-0.70.rst

```diff
@@ -29,8 +29,8 @@ write your queries without a ``FROM`` clause or use the ``VALUES`` syntax.
 Presto Verifier
 ---------------
 
-There is a new project, :doc:`/installation/verifier`, which can be used to
-verify a set of queries against two different clusters.
+There is a new project, `Presto Verifier <https://github.com/prestosql/presto/tree/master/presto-verifier>`_,
```

Comment:
Let's just call this "Presto Verifier", rather than linking to the source

Reply:
so just remove the link?